app/src/main/java/com/suntraining/aesport/ui/binding/ImageViewBindingAdapter.kt

```diff
@@ -0,0 +1,9 @@
+package com.suntraining.aesport.ui.binding
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.suntraining.aesport.utils.loadImage
+
+@BindingAdapter("app:image")
+fun loadImage(imageView: ImageView, url: String?) =
+    imageView.loadImage(url)
```

Comment:
remove, viết luôn trong Extension 
```suggestion
@BindingAdapter("app:imageUrl")
fun imageView.loadImage(url: String?)
```